poetry/layouts/layout.py

```diff
@@ -70,6 +76,30 @@ def __init__(
 
         self._author = author
 
+    @property
+    def basedir(self):  # type: () -> Path
+        return Path()
+
+    @property
+    def package_path(self):  # type: () -> Path
+        return self.basedir / self._package_path_relative
+
+    def get_package_include(self):  # type: () -> Optional[InlineTable]
```

Comment:
`InlineTable` should be in quotes, because it's imported only for type checking.